src/components/Tabs/Tabs.types.ts

```diff
@@ -29,4 +30,5 @@ export interface TabsProps extends SpacingProps {
   ) => boolean;
   onSelectTab?: (selectedValue: React.ReactText) => void;
   children: React.ReactNode[];
+  isExpanded: boolean;
```

Comment:
Should solve failing tests
```suggestion
  isExpanded?: boolean;
```